### PR TITLE
Two row-layout for CheckableItemSelectActivity (fix #14087)

### LINKED ISSUE #14087: Different layout for CheckableItemSelectActivity
### Describe your problem!

When opening the cache list info configuration for the first time, nothing is preselected. When leaving the activity again, the current state will be stored which leads to information being missing in CacheListActivity. 

### Reproducible

Yes

### c:geo Version

2023.03.12

### Additional Information

Two more things (unrelated to the main issue, sorry that I was to lazy to open separate issues):
- There could be a bit more margin at the left
- instead of having an "add" button for every element, it's IMHO more common to have two sections "selected items" and "available items" where all elements can be moved between those two sections.

### PR DESCRIPTION
**Pull request by moving-bits**
## Description
Use drag & drop to select entries (instead of using buttons) as suggested in #14087

### CODE REVIEW

**Comment by moving-bits** [comment]
retest this please

**Comment by moving-bits** [comment]
retest this please

File: main/src/main/res/layout/checkable_item_select_activity.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        </LinearLayout>
+</androidx.core.widget.NestedScrollView>
+
+    <!--<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (fm-sys) [inline main/src/main/res/layout/checkable_item_select_activity.xml:22]: Nice! Commented out code should be removed though...

Comment (moving-bits) [reply main/src/main/res/layout/checkable_item_select_activity.xml:22]: Sure, an is already locally. Currently I'm waiting for the CI tobe usable again (captcha problem on both executord), @mucek4 is already working on it.

**Pull request merged**